src/AppGenerator-Support/AppGeneratorAbstractGenerator.class.st

```diff
@@ -49,11 +50,12 @@ AppGeneratorAbstractGenerator >> fillWithDefaults [
 	properties at: #CommandLineHandler ifAbsentPut: 'appGeneratorExample'.
 
 	properties at: #VMIncludeZipFile ifAbsentPut: 'latest10.zip'.	
-	properties at: #VMZipFile ifAbsentPut: 'latest10.zip'.	
+	properties at: #VMZipFile ifAbsentPut: 'latest10.zip'.
+	properties at: #VMType ifAbsentPut: 'Spur'
```

Comment:
To keep the same behavior than before we should use

```suggestion
	properties at: #VMType ifAbsentPut: 'headless'
```

Reply:
The idea was to change the default :) (I think it is more common to have a default UI application than a headless one), but I will commit this change anyway as I need it now